spec/features/instructor_interface_spec.rb

```diff
@@ -52,6 +52,17 @@
     end
   end
 
+  describe "View Publishing Rights" do
```

Comment:
Hi everyone, please refer to point 5 under "On all refactoring" in the OSS project description. Login_as instructor 6 will fail since in test db there is no such user.